doc/src/parsing_idioms.md

```diff
@@ -80,7 +80,7 @@ A simple way of avoiding the complexity of the parameter's bounds is to encode
 it directly into the `Lexeme` type. A pleasant, and easily adapted way of doing
 this, is to semi-hard-code the value of `StorageT` into the AST, which we
 do by adding `type StorageT = u32;` to the programs part of the grammar
-(if [`CTParserBuilder::new_with_storaget`](https://docs.rs/lrpar/0.1.1/lrpar/ctbuilder/struct.CTParserBuilder.html#method.new_with_storaget)
+(if [`CTParserBuilder::new_with_storaget`](https://docs.rs/lrpar/0.1.1/lrpar/ctbuilder/struct.CTParserBuilder.html#method.new_with_storaget
```

Comment:
I still feel like there's an `or` missing here: `which we do by adding `type StorageT = u32;` to the programs part of the grammar (or if CTParserBuilder was used, substituting `u32` with whatever type was specified to that function).`